PRChecker/Shared/Unique Views/PullRequestCell.swift

```diff
@@ -9,7 +9,7 @@ import SwiftUI
 
 struct PullRequestCell: View {
     
-    @StateObject var pullRequest: PullRequest
+    let pullRequest: PullRequest
```

Comment:
Maybe this should be ObseveredObject so it refreshes?

Reply:
I think ObservedObject x4 also works, but I need to convert `PullRequest.Header/Content/Footer` from structure to class first for adopting `ObservableObject` protocol

Reply:
Ah that's true. We can probably skip that but I think it's worth it to do the full PR at least.

Reply:
I agree!